lib/server/game.js

```diff
@@ -27,6 +27,25 @@ class Game {
     return this.gameMode.maximumPlayerCount;
   }
 
+  get metadata() {
+    const players = [];
```

Comment:
I'd prefer using `Map` instead of `Object` in the first place. Maybe that refactoring could be done in a separate PR.